src/views/Admin.vue

```diff
@@ -360,6 +360,18 @@ export default {
     },
     userIds() {
       return Object.keys(this.users);
+    },
+    suggestedActivitesSorted() {
+      const array = Object.keys(this.activitySuggestions)
+        .map(suggestion => {
+          return {
+            name: suggestion,
+            count: this.activitySuggestions[suggestion].count
+          };
+        })
+        .sort((a, b) => b.count - a.count);
+      console.log(array);
```

Comment:
Do we need this log statement?

Reply:
no, we do not 👍 

good catch! fixing...

Reply:
Fixed!